tests/NLog.UnitTests/Targets/MailTargetTests.cs

```diff
@@ -664,6 +666,45 @@ public void MailTargetInitialize_WithoutSpecifiedSmtpServer_ThrowsConfigExceptio
             Assert.Throws<NLogConfigurationException>(() => mmt.Initialize(null));
         }
 
+        /// <summary>
+        /// Test for https://github.com/NLog/NLog/issues/690
+        /// </summary>
+        [Fact]
+        public void MailTarget_UseSystemNetMailSettings_False_Override()
+        {
+            var inConfigVal = @"C:\config";
+            var mmt = new MockMailTarget(inConfigVal)
+            {
+                SmtpServer = "",
```

Comment:
@304NotModified I thin testing that not both are mandatory is here. However I d have some issues letting the HostValue as `null`. And for the other scenario where PickUpDirectoryLocation is `null` any of the other tests took it into account.